archunit-tooling-test/src/test/java/com/tngtech/archunit/tooling/BaselineTest.java

```diff
@@ -0,0 +1,47 @@
+package com.tngtech.archunit.tooling;
+
+import com.tngtech.archunit.tooling.examples.RegularJUnit4Test;
+import com.tngtech.archunit.tooling.examples.RegularJUnit5Test;
+import org.junitpioneer.jupiter.cartesian.ArgumentSets;
+import org.junitpioneer.jupiter.cartesian.CartesianTest;
+import org.junitpioneer.jupiter.cartesian.CartesianTest.MethodFactory;
+
+import java.util.Map;
+import java.util.stream.Stream;
+
+public class BaselineTest extends BaseTest {
```

Comment:
This test isn't testing ArchUnit as such, it is only here to test the different test engines themselves